PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+** Don't forget to label this PR as either: Open for visibility or Ready for review! **
+
+JIRA: **(link to JIRA ticket)**
+Linked PRs: **(links to corresponding PRs, optional)**
+
+## Changes
+- (change1)
+
+## How to test-drive this PR
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
+
+## TODOS:
+- [ ] Tests have been written.
+- [ ] Tests are passing on your machine.
+- [ ] Change works in both Android and iOS.
+- [ ] CHANGELOG.md has been updated.
+- [ ] Scaffold is working. If a new feature was added, it was added to the Scaffold app.
+- [ ] +1 from an engineer on the Astro team.
```

Comment:
Should we have a step here as "[ ] Run the generator to make sure it still functions correctly" ?